src/hooks/useFirebase.js

```diff
@@ -1,16 +1,16 @@
 import { fb } from './../lib/firebase';
 
-export const useFirebase = (collection) => {
-  const db = fb.firestore().collection(collection);
+export const useFirebase = () => {
+  const db = fb.firestore().collection('lists');
 
   const getAll = () => db;
 
   const getById = (id) => {
     return db.doc(id).get();
   };
 
-  const create = (data) => {
-    return db.add(data);
+  const create = (userToken) => {
```

Comment:
what do you think if put this function just in one line :D

Reply:
done! also changed the other one line returns to inline :)